src/main/java/frc/team4909/robot/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -0,0 +1,37 @@
+package frc.team4909.robot.subsystems.elevator;
+
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.PWMTalonSRX;
+import edu.wpi.first.wpilibj.PWMVictorSPX;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class ElevatorSubsystem extends Subsystem{
+    PWMVictorSPX leftElevatorSPX, rightElevatorSPX;
+    PWMTalonSRX leftElevatorSRX, rightElevatorSRX, pivotSRX;
+    Encoder pivotEncoder, liftEncoder1, liftEncoder2;
+    DigitalInput topLimitSwitch, bottomLimitSwitch;
+
+    public void ElevatorSubsytem(){
+        //Intake Pivot
+        pivotSRX = new PWMTalonSRX(3);
+        pivotEncoder = new Encoder(1, 2);
+
+        //Lift
+        leftElevatorSPX = new PWMVictorSPX(4);
```

Comment:
Again, CAN, not PWM. The SPXs should be slaved. Also, see #52, we should use a separate file for constants and input/output IDs as we mentioned earlier.